.circleci/config.yml

```
@@ -9,13 +9,13 @@ default-test: &default-test
 
 defaults: &defaults
   docker:
-    - image: cimg/node:16.19
+    - image: cimg/node:20.19
   working_directory: ~/repo
 
 jobs: # a collection of steps
   test-node16:
```

```suggestion
  test-node:
```
and at line#90 please change the reference to

---

Yes! Nice catch, I did change it but something happened between the changes and it got lost.